src/NuGet.VisualStudio.Implementation/Extensibility/VsPackageUninstaller.cs

```diff
@@ -43,7 +49,12 @@ public void UninstallPackage(Project project, string packageId, bool removeDepen
 
             ThreadHelper.JoinableTaskFactory.Run(async delegate
```

Comment:
You may already know this, but, you have to rebase your branch with 3.0-rtm. For example, this particular line would be different on the 3.0-rtm branch